requirements/docker.txt

```diff
@@ -0,0 +1,37 @@
+appdirs==1.4.4
```

Comment:
Do we need all these requirements for the docker images?
Please review and remove if there are unnecessary libraries here.

Reply:
Yes `appdirs` and `setuptools_scm` not needed